importer/api/__init__.py

```diff
@@ -74,7 +74,7 @@ def import_course_from_path(path, repo_id, user_id):
     """
     bundle = XBundle()
     bundle.import_from_directory(path)
-    return import_course(bundle, repo_id, user_id)
+    import_course(bundle, repo_id, user_id)
```

Comment:
Just for a clean commit history.  i.e. git blame/history will show this was part of haystack, which could be confusing.  Generally the one commit, one PR, one logical change methodology.  So several small one or two line PRs are easy to review and merge and the history looks good, but larger lumbering ones get muddy.